app/src/js/applications/classifications/visualization/home-container.js

```diff
@@ -1,41 +1,36 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { PropTypes } from 'prop-types';
-import { connect } from 'react-redux';
+import { connect, useSelector } from 'react-redux';
 import ClassificationVisualization from './home';
 import { buildExtract, Loading } from '@inseefr/wilco';
 import loadClassification from 'js/actions/classifications/classification';
 import * as mainSelect from 'js/reducers';
 import * as select from 'js/reducers/classifications/classification';
-import { Stores } from 'bauhaus-utilities';
+import { Stores, Auth } from 'bauhaus-utilities';
 
 const extractId = buildExtract('id');
 
-class ClassificationVisualizationContainer extends Component {
-	constructor(props) {
-		super();
-	}
-	componentWillMount() {
-		const { classification, id } = this.props;
-		if (!classification) this.props.loadClassification(id);
-	}
-	componentWillReceiveProps({ id }) {
-		if (id !== this.props.id) {
-			this.props.loadClassification(id);
-		}
-	}
-	render() {
-		const { classification, id, secondLang, langs } = this.props;
-		if (!classification) return <Loading />;
-		return (
-			<ClassificationVisualization
-				classification={classification}
-				classificationId={id}
-				secondLang={secondLang}
-				langs={langs}
-			/>
-		);
+const ClassificationVisualizationContainer = (props) => {
+	const { classification, id, secondLang, langs } = props;
+	if (!classification) props.loadClassification(id);
```

Comment:
I think you should put this in a useEffect. 
Also, as we try to remove Redux, insead of calling loadClassification, can you make the HTTP request directly from this file ?